CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,5 @@
 # Code of Conduct
 
-By participating in this project, you agree to abide by the
-[thoughtbot code of conduct][tb-coc].
+By participating in this project, you agree to abide by the [thoughtbot code of conduct].
 
-[tb-coc]: https://thoughtbot.com/open-source-code-of-conduct
+[thoughtbot code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
```

Comment:
I've been treating Code of Conduct as a proper noun. Do you think that's worth keeping, or is it distracting?

Reply:
+1. I treat our Handbook and Guides as a proper noun, and I think by analogy I'd also treat our Code of Conduct as a proper noun. (But I wouldn't treat the idea of a code of conduct as a proper noun.)